diagrams/sequence-diagrams/creating-and-publishing-a-form.md

```diff
@@ -0,0 +1,28 @@
+# Creating and publishing a form
+
+```mermaid 
+sequenceDiagram
+  actor user as User
+  participant manager as Manager
+  participant designer as Designer
+  participant api as API
+
+  user -->> manager: Creates a new form
+  manager -->> api: Submit form creation information
+  api -->> manager: Returns new form information
```

Comment:
What new form information is being returned? Won't it just be `success` or `error`?